GoogleDataTransportCCTSupport/GDTCCTTestApp/viewcontroller.swift

```diff
@@ -39,6 +39,7 @@ public extension ViewController {
     ]
     testMessage.root.subMessage.repeatedSubMessage[0].samplingPercentage = 13.37
     event.dataObject = testMessage
+    event.customPrioritizationParams = ["needs_network_connection_info": true]
```

Comment:
Here, it is fine for "needs_network_connection_info" to just be a normal string since GDTCCT's API is entirely private.